src/trialscope.ts

```diff
@@ -268,26 +266,25 @@ export class TrialScopeQuery {
       }
     }
   }
-  // Get the mCODE filters as an array of strings
-  getmCODEFilters(): string[] {
-    const filterArray: string[] = [];
-    filterArray.push('primaryCancer: ' + this.mcode.primaryCancer);
-    filterArray.push('secondaryCancer: ' + this.mcode.secondaryCancer);
-    filterArray.push('histologyMorphology: ' + this.mcode.histologyMorphology);
-    filterArray.push('stage: ' + this.mcode.stage);
-    filterArray.push('tumorMarker: ' + this.mcode.tumorMarker);
-    filterArray.push('radiationProcedure: ' + this.mcode.radiationProcedure);
-    filterArray.push('surgicalProcedure: ' + this.mcode.surgicalProcedure);
-    filterArray.push('medicationStatement: ' + this.mcode.medicationStatement);
-    return filterArray;
+  /**
+   * Gets the set of mCode filters as an appropriate GraphQL block, such as
+   * "{primaryCancer: BREAST_CANCER}"
+   */
+  getMCODEFilters(): string {
```

Comment:
This will include age as an mcode filter, but since age hasn't actually been implemented within the trialscope mcode filter, it will cause the query to be returned as invalid.